demos/mri_reconstruction_demo/python/mri_reconstruction_demo.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+import numpy as np
+import cv2 as cv
+import argparse
+import time
+from openvino.inference_engine import IECore
+
+
+def kspace_to_image(kspace):
+    assert(len(kspace.shape) == 3 and kspace.shape[-1] == 2)
+    fft = cv.idft(kspace, flags=cv.DFT_SCALE)
+    img = cv.magnitude(fft[:, :, 0], fft[:, :, 1])
+    return cv.normalize(img, dst=None, alpha=255, beta=0, norm_type=cv.NORM_MINMAX, dtype=cv.CV_8U)
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='MRI reconstrution demo for network from https://github.com/rmsouza01/Hybrid-CS-Model-MRI (https://arxiv.org/abs/1810.12473)')
```

Comment:
not sure that it is good idea to put link on github repo in argparser (you add model to OMZ with demo, is not it?)